### PR TITLE
Do not assume IPv4 when disabling management

### PR DESCRIPTION
**Pull request by benjamreis**
See: https://github.com/xapi-project/xen-api/pull/6012

### CODE REVIEW

**Comment by benjamreis** [comment]
[v8.3-incoming scratch build](https://koji.xcp-ng.org/taskinfo?taskID=66315)

**Review by stormi — APPROVED** [review]

File: SPECS/xapi.spec [hunk 1]
```diff
@@ -77,6 +77,7 @@ Patch1006: xapi-24.19.2-update-new-fingerprint-fields-on-DB-upgrade.backport.pat
 Patch1007: xapi-24.19.2-more-fingerprint-field-updates-fixes.XCP-ng.patch
 Patch1008: xapi-24.19.2-ipv6-reset-networking.XCP-ng.patch
 Patch1009: xapi-24.19.2-keep-address-type-network-reset.XCP-ng.patch
+Patch1010: xapi-24.19.2-keep-ipv6-management-disable.XCP-ng.patch
```
Comment (stormi) [inline SPECS/xapi.spec:80]: Since it's already merged, you could add a comment stating which future version of XAPI is supposed to embed it.

**Review by gthvn1 — APPROVED** [review]

File: SPECS/xapi.spec (hunk 1 above)
Comment (benjamreis) [reply SPECS/xapi.spec:80]: I'll do it in next PR since everything have the been merged for the same version :+1:

**Comment by benjamreis** [comment]
[v8.3-incoming build](https://koji.xcp-ng.org/taskinfo?taskID=66318)

**Pull request merged**